raiden-dapp/src/views/TransferSteps.vue

```diff
@@ -0,0 +1,357 @@
+                {{ this.$t('transfer.steps.transfer.title') }}
+              </p>
+              <p class="transfer-steps__processing-transfer__description">
+                {{ this.$t('transfer.steps.transfer.description') }}
+              </p>
+            </div>
+          </v-stepper-content>
+        </v-stepper-items>
+      </v-stepper>
+    </v-row>
+
+    <stepper
+      :display="processingTransfer"
+      :steps="steps"
+      :done-step="doneStep"
+      :done="transferDone"
+    ></stepper>
+
+    <error-screen
+      :description="error"
+      :title="errorTitle"
+      :button-label="$t('transfer.error.button')"
+      @dismiss="error = ''"
+    ></error-screen>
+
+    <action-button
+      :enabled="continueBtnEnabled"
+      sticky
+      :text="$t(`transfer.steps.call-to-action.${step}`)"
+      @click="handleStep()"
+    >
+    </action-button>
+  </v-container>
+</template>
+
+<script lang="ts">
+import { Component, Mixins } from 'vue-property-decorator';
+import { RaidenPFS } from 'raiden-ts';
+import { BigNumber } from 'ethers/utils';
+
+import { BalanceUtils } from '@/utils/balance-utils';
+import { Token, Route, emptyDescription, StepDescription } from '@/model/types';
+import NavigationMixin from '@/mixins/navigation-mixin';
+import BlockieMixin from '@/mixins/blockie-mixin';
+import PathfindingServices from '@/components/PathfindingServices.vue';
+import FindRoutes from '@/components/FindRoutes.vue';
+import ActionButton from '@/components/ActionButton.vue';
+import Spinner from '@/components/Spinner.vue';
+import Stepper from '@/components/Stepper.vue';
+import ErrorScreen from '@/components/ErrorScreen.vue';
+
+@Component({
+  components: {
+    PathfindingServices,
+    ActionButton,
+    FindRoutes,
+    Spinner,
+    Stepper,
+    ErrorScreen
+  }
+})
+export default class TransferSteps extends Mixins(
+  BlockieMixin,
+  NavigationMixin
+) {
+  step: number = 1;
+  selectedPfs: RaidenPFS | null = null;
+  selectedRoute: Route | null = null;
+  pfsFeesConfirmed: boolean = false;
+  pfsFeesPaid: boolean = false;
+  mediationFeesConfirmed: boolean = false;
+  processingTransfer: boolean = false;
+  transferDone: boolean = false;
+  errorTitle: string = '';
+  error: string = '';
+  steps: StepDescription[] = [];
+  doneStep: StepDescription = emptyDescription();
+
+  convertToUnits = BalanceUtils.toUnits;
+
+  handleStep() {
+    if (this.step === 1 && this.selectedPfs && !this.pfsFeesConfirmed) {
+      this.pfsFeesConfirmed = true;
+      this.step = 2;
+      return;
+    }
+
+    if (this.step === 2 && this.selectedRoute) {
+      this.mediationFeesConfirmed = true;
+      this.step = 3;
+      return;
+    }
+
+    if (this.step === 3 && this.selectedRoute && this.selectedPfs) {
+      this.transfer();
+    }
+  }
+
+  get token(): Token {
+    const { token: address } = this.$route.params;
+    return this.$store.state.tokens[address] || ({ address } as Token);
```

Comment:
I noticed in my PR that when you have parameters in the URL and this causes an error during loading due to the fact that the default token we use doesn't have any decimals. 

It's not part of this PR but can you try refreshing the payment screen to verify that your don't get any console.errors?

Reply:
Thanks 🙏 
Sure will check it out tomorrow and create a follow up ticket if needed